torchbiggraph/converters/importers.py

```diff
@@ -72,7 +77,7 @@ def read(self, path: Path):
                 "'pip install parquet'"
             )
 
-        with path.open("rt") as tf:
+        with path.open("rb") as tf:
```

Comment:
Why did you change this to binary? Is it for speed? (If so leave a comment) Could there be any drawbacks? Do we ever read in non-ASCII files? Couldn't the name of the entities in FB15k or Wikidata contain Unicode characters? Note that strings that compare equal in Unicode aren't always bitwise identical (damned normalization).

Nit: my personal convention was, if opened in text mode (i.e., `rt`) call it `tf` (text file), if opened in binary mode (i.e., `rb`) call it `bf` (binary file). It helps me figure out what type I expect to get our of `.read()` (i.e., a `str` or a `bytes`) without having to track down where the `open` occurred.

Reply:
It was a bug, "rt" doesn't work; this is a binary file format.